RELEASE/scripts/autoscend/auto_post_adv.ash

```diff
@@ -642,19 +642,35 @@ void handlePostAdventure()
 		}
 
 		boolean doML = true;
+		boolean removeML = false;
+			// removeML MUST be true for purgeML to be used. This is only used for -ML locations like Smut Orc, and you must have 5+ SGEAs to use.
+			boolean purgeML = false;
+
 		if(get_property("kingLiberated").to_boolean())
 		{
 			doML = false;
 		}
 		if((equipped_amount($item[Space Trip Safety Headphones]) > 0) || (equipped_amount($item[Red Badge]) > 0))
 		{
 			doML = false;
+			removeML = true;
 		}
-		if(((get_property("flyeredML").to_int() > 9999) || get_property("auto_hippyInstead").to_boolean() || (get_property("auto_war") == "finished") || (get_property("sidequestArenaCompleted") != "none")) && (my_level() >= 13))
+		if(((get_property("flyeredML").to_int() > 9999) || get_property("auto_hippyInstead").to_boolean() || (get_property("auto_war") == "finished") || (get_property("sidequestArenaCompleted") != "none")) && ((my_level() >= 13) && (my_location() != $location[Oil Peak]) && (my_location() != $location[The Typical Tavern Cellar]) && (my_location() != $location[The Haunted Boiler Room])))
```

Comment:
try something like this instead:
```
boolean[location] highMLZones = $locations[Oil Peak, The Typical Tavern Cellar, The Haunted Boiler Room];
if(((get_property("flyeredML").to_int() > 9999) || get_property("auto_hippyInstead").to_boolean() || (get_property("auto_war") == "finished") || (get_property("sidequestArenaCompleted") != "none")) && ((my_level() >= 13) && !(highMLZones contains my_location())))
 ```
(I might have screwed up the parens on that if you blindly copy/paste)

Adds clarity, easier to extend in the future if we find need (and shortens the condition)

Reply:
Good idea. I added it, and also made a lowMLZones variable, just in case.

Also tried to organize better.